tensorflow/core/BUILD

```diff
@@ -1762,6 +1762,9 @@ tf_cc_tests(
         "graph/graph_test.cc",
         "graph/node_builder_test.cc",
         "graph/optimizer_cse_test.cc",
+# INTEL_MKL start
```

Comment:
I think what you'll want to do is: 

srcs = [
  ... current list of files ..
  ...
] + if_mkl(["graph/optimizer_merge_test.cc"]),